windows/container.go

```diff
@@ -136,7 +138,8 @@ func (c *container) Exec(ctx context.Context, opts containerd.ExecOpts) (contain
 		if err != nil {
 			log.G(ctx).Debug(err)
 		}
-		c.sendEvent(c.ctr.ID(), containerd.ExitEvent, p.Pid(), ec)
+		exitedAt := time.Now()
```

Comment:
should probably have `processExitCode` (see [here](https://github.com/containerd/containerd/blob/master/windows/hcs/process.go#L47)) save that timestamp since it's called in a separate go routine. It would be more accurate.